src/OpenTelemetry/OpenTelemetry.csproj

```diff
@@ -17,7 +17,11 @@
   <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.1' OR '$(TargetFramework)' == 'net6.0'">
     <RunApiCompat>false</RunApiCompat>
   </PropertyGroup>
-
+  
+  <ItemGroup Condition=" $(TargetFramework.StartsWith('net4')) ">
+    <Reference Include="System.Web" />
```

Comment:
we are very hesitant to adding dependencies to the core sdk packages, so this direction is a no-no.

Reply:
How would you like this handled then?

Reply:
Doesn't look like there is anyway to do this without using System.Web

https://github.com/dotnet/aspnetcore/issues/7400
https://github.com/dotnet/aspnetcore/issues/17069

Reply:
auto-instrumentation can bring this dependency (it's actually already there), it's easier for us since we are mixed to the application level. About libs brining extra dependencies, I agree with  @cijothomas .